client/src/index.jsx

```diff
@@ -111,8 +114,8 @@ const App = () => {
 
   return (
     <div>
-      <Overview productId={productId} styleId={styleId} averageStarRating={averageStarRating} totalNumberReviews={totalNumberReviews} productFeatures={productFeatures} updateSelectedProduct={updateSelectedProduct} myOutfit={myOutfit} setMyOutfit={setMyOutfit} reviewsRef={reviewsRef}/>
```

Comment:
Going to need reviewsRef={reviewsRef} to stay in here for Overview so the app doesn't break on that click -- super tiny edit